src/main/proto/buffer.proto

```diff
@@ -0,0 +1,23 @@
+/**
+    Copyright 2018 BarD Software s.r.o
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Author: Mikhail Shavkunov (@shavkunov)
+ */
+syntax = "proto3";
+
+option java_package = "com.bardsoftware.backend.fleet.rmarkdown";
+option java_outer_classname = "BufferProtocol";
+
+message Buffer {
```

Comment:
Please rename it to `CompilerFleetRequest`, the file to `fleet.proto` and outer class to `CompilerFleet`

Reply:
Done